Source/Extensions/Blazorise.DataGrid/DataGridSelectColumn.cs

```diff
@@ -37,6 +35,12 @@ protected internal override void SetValue( TItem item, object value )
     /// </summary>
     [Parameter] public Func<object, object> ValueField { get; set; }
 
+    /// <summary>
+    /// Provides additional information about the underlying type. This is useful when the bound data is not easily castable to the desired type.
+    /// I.e: enums
+    /// </summary>
+    [Parameter] public Type Type { get; set; }
```

Comment:
Rename to `ValueType`?

Reply:
Yea that seems fine, I was just playing with it, making it work. I don't remember if there's a name collision for `ValueType` though.